src/mono/netcore/sample/Android/AndroidSampleApp.csproj

```diff
@@ -36,6 +36,7 @@
         RuntimeIdentifier="$(RuntimeIdentifier)"
         SourceDir="$(OutputPath)"
         ProjectName="HelloAndroid"
+        EnableInterpreter="true"
```

Comment:
I wonder if we should call this `ForceInterpreter` instead since we're not just enabling the interpreter (e.g. mixed-mode style) but forcing it to be used for everything.

Reply:
Should that go all the way up?  `MonoEnableInterpreter` is the MSBuild prop to kick it all off.

Reply:
hmm it sounds clearer to me but if it means changing a lot of stuff unrelated to this then maybe it's not worth it.